src/Mobs/Monster.cpp

```diff
@@ -123,127 +120,20 @@ void cMonster::SpawnOn(cClientHandle & a_Client)
 
 
 
-bool cMonster::TickPathFinding(cChunk & a_Chunk)
+void cMonster::MoveToWayPoint(cChunk & a_Chunk)
 {
-	if (!m_PathfinderActivated)
-	{
-		return false;
-	}
-	if (m_TicksSinceLastPathReset < 1000)
-	{
-		// No need to count beyond 1000. 1000 is arbitary here.
-		++m_TicksSinceLastPathReset;
-	}
-
-	if (ReachedFinalDestination())
+	if ((m_NextWayPointPosition - GetPosition()).Length() < GetWidth()/2)
```

Comment:
The convention says "parenthesis around comparisons", not "parenthesis around comparison operands"
Similarly, we write `if (a > b + 2)` instead of `if (a > (b + 2))`